apps/che/src/main/fabric8/deployment.yml

```diff
@@ -144,13 +144,17 @@ spec:
         imagePullPolicy: "IfNotPresent"
         name: che
         readinessProbe:
-          tcpSocket:
+          httpGet:
+            path: /api/system/state
             port: 8080
-          initialDelaySeconds: 20
+            scheme: HTTP
+          initialDelaySeconds: 60
```

Comment:
@l0rd if you remove readiness check it skip readiness check and try to call endpoint but it fails on 504 because there is not server. 504 is caused because there is no server running (yet), but 503 is caused because there is no service (deployed war on server) running yet.

Ad optimization: why it is changed then? This PR changes also the timeout then we would create another one to change it to value at the point before this PR? It does not make sense to me.